README.md

```diff
@@ -91,45 +91,64 @@ houses the portion of the workflow that does estimation
 of the DFE.
 
 
-------------------
-
-A small example to simulate genomes with selection is described below.
-
-
-# Simulating selection with stdpopsim
-
-For each HomSap chromosome one can simulate selection in specific regions (e.g. coding region)
-by using the function in stdpopsim and passed to
-`stdpopsim.species.get_annotations` (see [the catalog](https://popsim-consortium.github.io/stdpopsim-docs/latest/tutorial.html#simulating-selection-on-exons) for details). Below is a complete example
-that uses a given annotation to simulate selection on chr20.
-
-```python
-import stdpopsim
+## Running the workflow on a cluster
```

Comment:
I ran into an issue compiling due to my LD_LIBRARY_PATH

Should export `LD_LIBRARY_PATH=""` be included somewhere in the pipeline?

Reply:
probably not as that will differ per user environment?